### PR TITLE
#4 move dependency installation to Docker image build-time

### PR DESCRIPTION
**Pull request by con-mark**
https://github.com/dadwalakshay/the-django-boilerplate/issues/4

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,4 +12,6 @@ COPY . /code
 
 RUN pip install --no-cache pipenv==2020.11.15
 
+RUN pipenv install --system --deploy
```
Comment (dadwalakshay-ai) [inline Dockerfile:15]: remove the same from start.sh, there'll be a redundant installation of dependencies

**Review by dadwalakshay-ai — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (xdadwal) [inline Dockerfile:15]: remove the same from start.sh, else there will be a redundant installation of dependencies.

**Review by xdadwal — APPROVED** [review]

**Review by xdadwal — CHANGES_REQUESTED** [review]
Please do the required changes.

**Comment by con-mark** [comment]
Hi @dadwalakshay,

I have pushed the changes now didn't save correctly before pushing previously :)

Let me know if anything else needed.

**Review by xdadwal — APPROVED** [review]

**Comment by xdadwal** [comment]
Thanks for contributing, approved your PR.

**Pull request merged**